src/commands/utils/target/read.ts

```diff
@@ -0,0 +1,31 @@
+import {Command, Flags} from '@oclif/core'
+import {ExecJSON} from 'inspecjs'
+import fs from 'fs'
+
+export default class ReadTarget extends Command {
+    static usage = 'target read -i <hdf-json> [-o <target-json>]'
+
+    static description = 'Read the `target` attribute in a given Heimdall Data Format JSON file and send it to stdout or write it to a file'
+
+    static examples = ['saf target read -i hdf.json -o target.json']
+
+    static flags = {
+      help: Flags.help({char: 'h'}),
+      input: Flags.string({char: 'i', required: true, description: 'An input HDF file'}),
+      output: Flags.string({char: 'o', description: 'An output `target` JSON file (otherwise the data is sent to stdout)'}),
+    }
+
+    async run() {
+      const {flags} = await this.parse(ReadTarget)
+
+      const input: ExecJSON.Execution & {target?: unknown} = JSON.parse(fs.readFileSync(flags.input, 'utf8'))
+
+      const target = input.target || {}
```

Comment:
I didn't realize that `target` and `passthrough` were distinct fields.

Reply:
Yes, passthrough is just a grab bag for data that we don't really have a better place to store whereas target is intended on being identifying information for a run or the environment where the scan happened.

Reply:
Thanks for the clarification. I am guessing that if users need to know that, it would be in the schema documentation or something, right? Just making sure

Reply:
As of right now, we don't have passthrough nor target in the schema so they are not documented.  @ejaronne is wanting to "soft release" this functionality as continue to talk with Chef about formally putting it in their schema.